commons/utils.js

```diff
@@ -0,0 +1,60 @@
+const { BRIDGE_MODES, FEE_MANAGER_MODE } = require('./constants')
+
+function decodeBridgeMode(bridgeModeHash) {
+  switch (bridgeModeHash) {
+    case '0x92a8d7fe':
+      return BRIDGE_MODES.NATIVE_TO_ERC
+    case '0xba4690f5':
+      return BRIDGE_MODES.ERC_TO_ERC
+    case '0x18762d46':
+      return BRIDGE_MODES.ERC_TO_NATIVE
+    default:
+      throw new Error(`Unrecognized bridge mode hash: '${bridgeModeHash}'`)
+  }
+}
+
+const decodeFeeManagerMode = managerModeHash => {
```

Comment:
Don't think that getting the fee manager mode is used anywhere besides the UI. @patitonar please confirm.
Does it really make sense to move it to common part of the repo?

Reply:
It's true that it is used only in UI, but on the other hand it feels like it should be in the same place as `decodeBridgeMode` since they're very similar. If you think otherwise then I'll remove it and keep this in UI.

Reply:
Yes, it is used only in UI, but I agree that it makes sense to keep this method together with other decoding methods, so I'm OK to moving it to common

Reply:
great! I am approving the PR